core/src/test/java/com/amplifyframework/logging/AndroidLoggingPluginTest.java

```diff
@@ -129,6 +135,36 @@ public void noContentLoggedAtThresholdNone() {
         assertTrue(systemLog.getLines().isEmpty());
     }
 
+    /**
+     * The {@link AndroidLoggingPlugin#forCategory(CategoryType)} implementation contains
+     * special rules to normalize a log slut from a category name. Validate these conversions.
+     */
+    @Test
+    public void correctSlugGeneratedForCategories() {
+        AndroidLoggingPlugin plugin = new AndroidLoggingPlugin();
+
+        final Map<CategoryType, String> actuals = new HashMap<>();
+        for (CategoryType categoryType : CategoryType.values()) {
+            String result = plugin.forCategory(categoryType).getNamespace();
+            actuals.put(categoryType, result);
+        }
+        final Map<CategoryType, String> expected = new HashMap<>();
+        expected.put(CategoryType.ANALYTICS, "amplify:analytics");
+        expected.put(CategoryType.API, "amplify:api");
+        expected.put(CategoryType.AUTH, "amplify:auth");
+        expected.put(CategoryType.DATASTORE, "amplify:data-store");
```

Comment:
Huh. I guess it stands to reason that `DataStore` would lowercase to `data-store`, but I hadn't seen it before.

Reply:
Hm, yea, same. `datastore` looks more natural. I can probably just update the logic to print a string of lowercase letters without punctuation, that might work.